packages/@react-spectrum/s2/src/SegmentedControl.tsx

```diff
@@ -32,16 +32,22 @@ export interface SegmentedControlProps extends Omit<RadioGroupProps, 'isReadOnly
    * Whether the segmented control is disabled.
    */
   isDisabled?: boolean,
-  /**
-   * Defines a string value that labels the current element.
-   */
-  'aria-label': string
+  /** The id of the currently selected item (controlled). */
+  selectedKey?: string | null,
+  /** The id of the initial selected item (uncontrolled). */
+  defaultSelectedKey?: string,
+  /** Handler that is called when the selection changes. */
+  onSelectionChange?: (id: string) => void
 }
-export interface SegmentedControlItemProps extends Omit<RadioProps, 'children' | 'className' | 'style' | 'onHoverStart' | 'onHoverEnd' | 'onHoverChange'>, StyleProps {
+export interface SegmentedControlItemProps extends AriaLabelingProps, StyleProps {
   /**
-   * The content to display in the control item.
+   * The content to display in the segmented control item.
    */
-  children: ReactNode
+  children: ReactNode,
+  /** The id of the item, matching the value used in SegmentedControl's `selectedKey` prop. */
+  id: string,
```

Comment:
Are we ok requiring id?

Reply:
well... Radio requires `value` so I think we have to for now. Since this is mostly used for view switching, I imagine 99% of use cases will be controlled anyway.

Reply:
we have to for this, can't generate defaults since it's not a collection and the top component doesn't know all the ids